index.html

```diff
@@ -160,11 +160,11 @@ <h2 class="cntct-ttl">Contact me</h2>
             <p class="cntct-info">If you have an application you are interested in developing, a feature that you need built
               or a project that needs coding. I'd love to help with it</p>
             </div>
-            <form name="contact-form" accept-charset="utf-8" action="https://formspree.io/f/mwkjpbkq"   method="post">
+            <form name="contact-form" accept-charset="utf-8" action="https://formspree.io/f/mwkjpbkq"   method="post" id="contactForm">
               <div class="frm-inputs">
                 <input class="frm-tgs" type="text" name="name" id="full-name" placeholder="First & Last Name " required
                 maxlength="30">
-                <input class="frm-tgs" type="email" name="_replyto" id="email-address" placeholder="email@domain.com"
+                <input class="frm-tgs" type="email" name="email" id="email-address" placeholder="email@domain.com"
                 required>
                 <textarea class="frm-tgs" rows="5" name="message" placeholder="Leave your message here." required
                 maxlength="500"></textarea>
```

Comment:
- [ ] I noticed that If a user submits the contact form with the email field containing uppercase characters an error message shows up but when a user submits the contact form with the email field containing only lowercase characters the form is not sent normally as it should. However, this is a part of the project requirements so kindly fix this :+1:
- I have attached a video below for your reference :100:
[Screencast from 02-01-2023 11:02:14 PM.webm](https://user-images.githubusercontent.com/102699958/216186717-f0cd0e74-d48c-4039-ab9c-61b236319b68.webm)
- [ ] Project Requirement
![image](https://user-images.githubusercontent.com/102699958/216187072-517dcdba-b3c5-4df4-9f3b-4bcb8d5dbd4f.png)